script/script.js

```diff
@@ -0,0 +1,196 @@
+import { desktopProjects, mobileProjects } from './data.js';
+
+const hamburger = document.querySelector('#hamburger');
+const sideMenu = document.querySelector('#side-menu');
+const cross = document.querySelector('#cross');
+const body = document.querySelector('body');
+const menuLinks = document.querySelectorAll('.menu-link');
+const projectContainer = document.querySelector('#portfolio');
+const desktopContainer = document.querySelector('#desktop-portfolio');
+const modalTitle = document.querySelector('.modal-title');
+const modalCompany = document.querySelector('.modal-company');
+const modalCounterText = document.querySelector('.modal-countertext');
+const modalYear = document.querySelector('.modal-year');
+const modalIMage = document.querySelector('#modal-image');
+const modalDescription = document.querySelector('#modal-description');
+const modalSkills = document.querySelector('.modal-skills');
+const modalSeeLive = document.querySelector('#see-live');
+const modalSeeSource = document.querySelector('#see-source');
+const modal = document.querySelector('.modal');
+const modalContainer = document.querySelector('.modal-container');
+
+const openSideMenu = () => {
+  sideMenu.classList.add('open-side-menu');
+  body.style.overflow = 'hidden';
+};
+
+const closeSideMenu = () => {
+  sideMenu.classList.remove('open-side-menu');
+  body.style.overflow = 'unset';
+};
+
+hamburger.addEventListener('click', openSideMenu);
+cross.addEventListener('click', closeSideMenu);
+
+for (let i = 0; i < menuLinks.length; i += 1) {
+  menuLinks[i].addEventListener('click', closeSideMenu);
+}
+
+// popup
+// Mobile Version//
+for (let i = 0; i < mobileProjects.length; i += 1) {
+  const data = mobileProjects[i];
+  const card = document.createElement('div');
+  card.classList.add('card');
+
+  card.innerHTML = `
+  <img class="card-image" src=${data.cardImage1} alt="mobileVersion" />
+  <div class="tonic-area">
+    <h2 class="cards-details">${data.cardTitle}</h2>
+    <div class="highlights">
+      <span class="highlight-text">${data.company}</span>
+      <img src="images/Counter.png" alt="Counter.png" />
+      <span class="countertext">${data.counterText}</span>
+      <img src="images/Counter.png" alt="Counter.png" />
+      <span class="year">${data.year}</span>
+    </div>
+    <article class="cards-description">
+      <p>
+        ${data.description}
+      </p>
+    </article>
+    <ul class="skills">
+      <li id="html">html</li>
+      <li id="css">css</li>
+      <li id="javascript">javascript</li>
+    </ul>
```

Comment:
- [ ] Please make sure that you use the Javascript map function to populate the project's technologies from the array of each project technology so that if you have 10 projects technologies, they will be populated and make it easy to add a project to your portfolio :pray: